### PR TITLE
recache page after mutation

### PR DESCRIPTION
**Pull request by amaliwanag2**
@kmjennison I *think* we still want a stale while revalidating strategy to cache assets besidies the /newtab page but I left a comment in the workbox config with more details

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ module.exports = {
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'error',
     'no-console': 'error',
+    'no-unused-expressions': ['error', { allowShortCircuit: true }],
```
Comment (kmjennison) [inline .eslintrc.js:22]: Let's not disable this. I agree with [this criticism](https://github.com/airbnb/javascript/issues/963#issuecomment-235048756) of readability.

File: src/utils/mutations/callMutation.js [hunk 2]
```diff
@@ -36,10 +36,12 @@ const callMutation = async ({ mutation, variables }) => {
   // creating the Relay environment.
   const environment = getRelayEnvironment()
 
-  return commitMutation(environment, {
+  const result = await commitMutation(environment, {
     mutation,
     variables,
   })
+  cache && recachePage()
```
Comment (kmjennison) [inline src/utils/mutations/callMutation.js:43]: Use `if(cache) { ... }`

File: next.config.js [hunk 3]
```diff
@@ -160,6 +160,10 @@ const nextConfig = {
         // variants with our base path. Note that our base path, "/newtab", is
         // hardcoded here.
         // https://regex101.com/r/5cs6L7/1/tests
+        // see caching strategies here https://developers.google.com/web/tools/workbox/modules/workbox-strategies
+        // I think we still want stale while revalidating although maybe we want cache only?
+        // "The cache only strategy ensures that responses are obtained from a cache. 
+        // This is less common in workbox, but can be useful if you have your own precaching step.""
```
Comment (kmjennison) [inline next.config.js:166]: Yes, let's just leave this for now. If we did cache only, we'd want it only for the `newtab` resource, and that's just an optimization that could cause a bug of serving stale resources (for example, if we modify/disable the recaching mutation logic).

Go ahead and remove comments.

**Review by kmjennison — CHANGES_REQUESTED** [review]
Nice work!

File: next.config.js [hunk 4]
```diff
@@ -160,6 +160,7 @@ const nextConfig = {
         // variants with our base path. Note that our base path, "/newtab", is
         // hardcoded here.
         // https://regex101.com/r/5cs6L7/1/tests
+        // see caching strategies here https://developers.google.com/web/tools/workbox/modules/workbox-strategies
```
Comment (amaliwanag2) [inline next.config.js:163]: still seems useful to have this one line

**Review by kmjennison — APPROVED** [review]
LGTM 🍝

**Pull request merged**